### PR TITLE
Update by-laws.md

### PR DESCRIPTION
**Pull request by mrhinkle**
Update of the Node.js Foundation bylaws as of 12-05-16.

### CODE REVIEW

**Comment by nebrius** [comment]
Thanks for adding these, it's great to see them in the repo! The formatting looks good to me, but I'll leave it to the other board members to approve, as I don't have the context/insight to comment on anything except formatting :)

**Review by williamkapke — CHANGES_REQUESTED** [review]
I'm unclear what is actually changing in this PR. Seems like maybe it is just the formatting of the incorrect bylaws? The items listed in #53 are _not_ in here.

I'm pretty sure the PRs I have waiting for approval (#60 and #61) already covers all the correct changes needed. Can you comment on those with why we should go with this version instead?

**Comment by williamkapke** [comment]
Please work with the existing issues/PRs.

**Comment by jasnell** [comment]
I actually prefer this over the others but agree that a summary of the changes would be helpful. I think this PR should be reopened and @mrhinkle given more of an opportunity to respond to feedback.

File: by-laws.md
Comment (jasnell) [inline by-laws.md:6]: @mrhinkle ... from a formatting point of view, it would be significantly easier to review if there were whitespace between the headers and paragraphs... e.g.

```
### Section 1.1 Name

The name of the ...

```

File: by-laws.md
Comment (jasnell) [inline by-laws.md:20]: Should `not for profit` be hyphenated? i.e. `not-for-profit`

**Pull request closed without merging**
